src/Options/index.js

```diff
@@ -577,4 +577,8 @@ export interface LogLevels {
   :DEFAULT: error
   */
   triggerBeforeError: ?string;
+  /* Log level used by the Cloud Code Functions. Default is `info`.
+  :DEFAULT: info
+  */
+  cloudFunctionRan: ?string;
```

Comment:
The existing log level that are used by triggers are differentiated by success or error. If I understand this correctly, then we should do the same for Cloud Functions. Otherwise it's unclear whether the log level set here is used for success or for error logging.

```suggestion
  cloudFunctionSuccess: ?string;
```

If you want to could also add the log level `cloudFunctionError` to this PR and use that option in the lines below where you see a failed Cloud Function execution being caught:

> logger.error(`Failed running cloud function ${functionName} for user ${userString} with:\n  Input: ${cleanInput}\n  Error: ` +